crates/volta-core/src/tool/package/mod.rs

```diff
@@ -132,25 +135,27 @@ pub fn uninstall(name: &str) -> Fallible<()> {
 
     // if the package config file exists, use that to remove any installed bins and shims
     let package_config_file = home.default_package_config_file(name);
-    let package_found = if package_config_file.exists() {
-        let package_config = PackageConfig::from_file(&package_config_file)?;
 
-        for bin_name in package_config.bins {
-            remove_config_and_shim(&bin_name, name)?;
-        }
-
-        remove_file_if_exists(package_config_file)?;
-        true
-    } else {
-        // there is no package config - check for orphaned binaries
+    let remove_orphan_binaries_if_any = || -> Fallible<bool> {
```

Comment:
Since we aren't re-using this function anywhere else, I think it would be slightly clearer if this block was included in the `match` arm below, instead of creating a closure and then calling it.